Fix a little typo in Announcing Hanami v2.0.0.alpha2 (#523)
hanami/hanami.github.io: source/blog/2021-05-04-announcing-hanami-200alpha2.html.markdown

- A **completely rewritten application core**, offering advanced application-level state management and code loading capabilities
- An **always-there auto-injection mixin**, making it easy to model your behavior as functional, composable objects

> - Built-in **application settings** that provide first-class support

Reviewer: ```suggestion
- Built-in **application settings** with first-class support
```
